zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/RoleControllerTest.java

```diff
@@ -46,8 +47,11 @@ void setup() {
   @Test
   void createRoleShouldReturnCreated() {
     // given
+    final var roleId = "testRoleId";
```

Comment:
🔧 Can you make this test a `ParameterizedTest` ? 

🔧 Please add a test to verify that the request is rejected if the id is too long/invalid